charts/cluster-api-runtime-extensions-nutanix/addons/cni/cilium/values-template.yaml

```
@@ -36,8 +36,15 @@ envoy:
 {{- with .ControlPlane }}
 {{- range $key, $val := .metadata.annotations }}
 {{- if eq $key "controlplane.cluster.x-k8s.io/skip-kube-proxy" }}
+# Although its safe to set k8sServiceHost in all cases,
+# we're keeping it here to force a Cilium Pod rollout when enabling the kubeProxyReplacement feature.
 k8sServiceHost: auto
 kubeProxyReplacement: true{{ break }}
+routingMode: tunnel
+tunnelProtocol: geneve
+loadBalancer:
+  mode: dsr
+  dsrDispatch: geneve
```

Will need to be before the break
```suggestion
k8sServiceHost: auto
kubeProxyReplacement: true
routingMode: tunnel
tunnelProtocol: geneve
loadBalancer:
  mode: dsr
  dsrDispatch: geneve
{{- break }}
```

---

TY!